src/Mod/PartDesign/App/FeatureHole.cpp

```diff
@@ -1366,8 +1397,8 @@ App::DocumentObjectExecReturn *Hole::execute(void)
             mkWire.Add(BRepBuilderAPI_MakeEdge(lastPoint, newPoint));
             lastPoint = newPoint;

```

Comment:
Is it correct to have these lines here? They look like the tip formation code that's now in the conditional above.

Reply:
Thanks, this was a leftover from copy paste (doubled code). It should not be there. Fixed now.